includes/ucf-post-list-common.php

```diff
@@ -17,23 +17,23 @@ class UCF_Post_List_Common {
 		 * @param $title string | a title string to display within the post list markup
 		 * @return string | post list HTML string
 		 **/
-		public static function display_post_list( $items, $layout, $title ) {
+		public static function display_post_list( $posts, $layout, $show_image, $posts_per_row, $list_title ) {
```

Comment:
If the `$show_image`, `$post_per_row` or `$list_title` variables are unique to the new card layout, I would recommend passing them in through an argument array instead of extending the base function. So: `function display_post_list( $posts, $layout, $args ) { }`, and then handle the layout specific arguments via the arg array in the layout specific template functions. Or if `$list_title` is supposed to be available on all layouts, then `($posts, $layout, $list_title, $args)`. I'm not sure how that's going to fit into the option paradigm you all have going on with this plugin where all the defaults can be set in the config, but this I think will give the base code more extensibility without us having to go in and update code to accept more arguments.